app/controllers/baskets_controller.rb

```diff
@@ -1,6 +1,6 @@
 class BasketsController < ApplicationController
   def index
-    @baskets = Basket.all
+    @baskets = current_user.baskets.all
```

Comment:
I think you don't need the `all` 🤔

Reply:
Yes 😅 you're right already got rid of it in later commits